src/components/elements/table/tableRowActionUtil.tsx

```diff
@@ -25,6 +25,7 @@ export const BASE_ACTION_COLUMN_DEF: ColumnDef<any> = {
   render: '', // gets overridden when used
   sticky: 'right',
   width: '0px',
```

Comment:
Suggest passing `whiteSpace: 'nowrap'` into the sx, and width `1%` so that the action column takes up the least amount of space it needs. I think this makes it less distracting to the table especially when its only the icon button.

Reply:
Sounds good, I'll also reduce the x padding.